### ISSUE TITLE
312156859 Signup allowed in a county not accepting transcribers (Sandra)

### ISSUE THREAD

**Issue by FreeREGcomputer** [FreeUKGen/FreeUKRegProductIssues#1553]
Issue reported by **FR Exec Lead** at 2024-11-21 22:14:19 UTC
Time: 2024-11-21T22:03:26+00:00
Session ID: f54de04cb2059295cc740a2fd6b1d1cc
Problem Page URL: [/userid_details/672b720db18f7c43da136f31?locale=en&page_name=](/userid_details/672b720db18f7c43da136f31?locale=en&page_name=)
Previous Page URL: [https://www.freereg.org.uk/userid_details/672b720db18f7c43da136f31/edit?locale=en](https://www.freereg.org.uk/userid_details/672b720db18f7c43da136f31/edit?locale=en)
Reported Issue:
A transcriber was able to sign up in a county that was not accepting transcribers. I was approached by the training coord to transfer the transcriber for training and was told she was in the anywhere syndicate, but when I went to find her she was in the ROE syndicate, whose coord had no idea of this happening. Can we look into this please
https://www.freereg.org.uk/uploads/feedback/screenshot/673fb0bbb18f7c5a16a07d45/LanaH_screenshot.jpg    

![Screenshot](https://www.freereg.org.uk)

**Comment by Sandra-AW** [comment]
![image](https://github.com/user-attachments/assets/75abd3c9-0ecf-4416-82aa-0402cfd0a8f7)

**Comment by Sandra-AW** [comment]
Here is a copy of the email I received showing the signup for userid lanahamilton as noted above and also one other for Stephen79 to the ROE syndicate on the same day. Something went wrong when these people signed up.
![image](https://github.com/user-attachments/assets/5ade2ff1-e424-4265-8f25-82fa608478cc)

**Comment by Vino-S** [comment]
@stsccfr 
This change is deployed to LIVE. Could you give it a test, please?

**Comment by stsccfr** [comment]
I don't know how to test this.  When a transcriber signs up, they have to pick the Syndicate from a list, and that list only contains syndicates that are currently recruiting.  Sandra's attached screenshot is a dead link (we need to fix this within GitHub somehow) so I can't see what she was seeing, but she says that somehow the transcriber was in ROE, but how did that happen?  If I try to sign up as a transcriber, I don't see ROE in the list of available syndicates, nor do I see the Anywhere syndicate.

**Comment by stsccfr** [comment]
I should add that there is another little bug associated with this.  In the Info popup help associated with the "Syndicates available" list on the volunteer signup page, it says "If you are in doubt, select the 'England, rest of' group" but there is no such option in the list.

**Comment by Vino-S** [comment]
Create userid option should not be available for Non sys admins
<img width="595" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOVdpQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3172add91bdc1cd453a75d01ed24881dfc71ce14/Screenshot%20from%202025-01-29%2014-23-53.png" alt="Screenshot from 2025-01-29 14-23-53.png" />

**Comment by Vino-S** [comment]
@Vino-S to remove the back-up sentence from the Syndicate dropdown in the transcriber sign-up

**Comment by Vino-S** [comment]
Change Move to inactive syndicate to `Move to To Be Destroyed Syndicate`

**Comment by Vino-S** [comment]
County coord is able to access the Create Userid Action from the Manage Syndicate action. Needs to be removed

**Comment by Vino-S** [comment]
Hi @stsccfr 
I have updated the changes. Create Userid should only be available when the current role is sys_admin.
Both CC and SC should not have this action

**Comment by stsccfr** [comment]
I no longer see the Create Userid when I log in as a Coordinator, so that much is fixed now.  However, on the Volunteer sign-up page, we still need to remove the reference to "England, Rest of" from the Information help popup that is associated with the Syndicates Available list.  The popup should read: "Select a Syndicate to work with. You can move later depending upon your interests."

**Comment by DeniseColbert** [comment]
Ready for deployment

**Comment by Vino-S** [comment]
Complete, closing

**Comment by stsccfr** [comment]
Did all of the code get deployed here?  I just visited the Volunteer Signup page and the informational help for the list of Available Syndicates is still telling users to select "England, rest of" but that particular syndicate is not in the list.  I remember discussing this in a scrum and I thought a code change had been made for it.

**Comment by Vino-S** [comment]
Code deployed

**Comment by stsccfr** [comment]
Yes, the information message now shown on the Live signup page says simply "Select a syndicate to work with." so I'm closing this case.
